integrationtests/src/test/java/org/axonframework/integrationtests/commandhandling/AbstractAggregateMemberTest.java

```diff
@@ -0,0 +1,233 @@
+package org.axonframework.integrationtests.commandhandling;
```

Comment:
I'm missing a copyright notice in this file.
You can copy-paste it from another file, or add the [copyright notice template](https://github.com/AxonFramework/AxonFramework/blob/master/copyright-template.xml) to your IDE.